src/coreclr/pal/src/libunwind/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 # It overwrites the one found in upstream
 project(unwind)
 
+# Disable C11 compat
+unset(CMAKE_C_STANDARD)
+unset(CMAKE_C_STANDARD_REQUIRED)
```

Comment:
libunwind is a C library, I think setting `project(unwind C)` on line 3 would be a more appropriate fix. cc @janvorli

Reply:
similar to https://github.com/dotnet/runtime/blob/03059a96c73171435ed074e4fa81108eb0cd23a1/src/libraries/Native/Windows/System.IO.Compression.Native/zlib/CMakeLists.txt#L4 and many other examples in the repo.